src/main/resources/messages_it.yml

```diff
@@ -1,41 +1,43 @@
-reg_msg: '&cPerfavore, per connetterti al server devi prima registrarti con il comando: "/register <password> <ConfermaPassword>"'
-reg_email_msg: '&cPerfavore, per connetterti al server devi prima registrarti con il comando: "/register <email> <confermaEmail>"'
+login_msg: '&cPerfavore, effettua l''autenticazione con il comando: "/login <password>"'
+reg_msg: '&cPerfavore, esegui la registrazione con il comando: "/register <password> <confermaPassword>"'
+reg_email_msg: '&cPerfavore, esegui la registrazione con il comando: "/register <email> <confermaEmail>"'
 usage_unreg: '&cUtilizzo: /unregister <password>'
 pwd_changed: '&cPassword cambiata con successo!'
-user_unknown: "&cL'utente non ha ancora eseguito la registrazione."
+user_unknown: '&cL''utente non ha ancora eseguito la registrazione.'
 password_error: 'Le Password non corrispondono!'
-invalid_session: "I tuoi dati di connessione attuali non sono quelli utilizzati in precedenza. Attendere la fine della sessione attuale."
+password_error_nick: 'Non puoi usare il tuo nome utente come password, scegline un''altra!'
+password_error_unsafe: 'La password che hai inserito, scegline un''altra!'
+invalid_session: 'I tuoi dati di connessione attuali non sono quelli utilizzati in precedenza. Attendi la fine della sessione attuale.'
 reg_only: 'La registrazione è effettuabile soltanto attraverso il sito web! Perfavore, vai su http://example.com per procedere!'
-logged_in: '&cHai già eseguito l'autenticazione!'
+logged_in: '&cHai già eseguito l''autenticazione!'
 logout: '&cDisconnessione avvenuta correttamente!'
 same_nick: 'Lo stesso nickname è già online sul server!'
 registered: '&cRegistrato correttamente!'
 pass_len: 'La tua password è troppo corta o troppo lunga, prova a cambiarla!'
 reload: 'La configurazione e il database sono stati ricaricati con successo!'
-timeout: "Tempo scaduto per effettuare l'autenticazione"
+timeout: 'Tempo scaduto per effettuare l''autenticazione'
 usage_changepassword: 'Utilizzo: /changepassword <vecchiaPassword> <nuovaPassword>'
 name_len: '&cIl tuo nickname è troppo corto o troppo lungo!'
 regex: '&cIl tuo nickname contiene caratteri non consentiti. I caratteri consentiti sono: REG_EX'
-add_email: '&cPer poter recuperare la password in futuro, aggiungi una email al tuo account con il comando: "/email add tuaEmail confermaEmail"'
-bad_database_email: 'Il comando /email è utilizzabile solo con database MySQL o SQLite, riporta questo errore ad un Admin!'
+add_email: '&cPer poter recuperare la password in futuro, aggiungi una email al tuo account con il comando: "/email add <tuaEmail> <confermaEmail>"'
+bad_database_email: 'Il comando /email è utilizzabile solo con un database MySQL o SQLite, riporta questo errore ad un Admin!'
 recovery_email: '&cHai dimenticato la tua password? Puoi recuperarla eseguendo il comando: "/email recovery <tuaEmail>"'
-usage_captcha: '&cUtilizzo: /captcha <ilCaptcha>'
+usage_captcha: '&cUtilizzo: /captcha THE_CAPTCHA'
```

Comment:
La stringa "THE_CAPTCHA" o anche "<theCaptcha" è un placeholder che viene sostituito dal Captcha in se. Me ne sono accorto dal mio server e l'ho confermato ieri, spulciando fra i codici